multicluster/hack/update-codegen-dockerized.sh

```diff
@@ -60,9 +103,9 @@ $GOPATH/bin/informer-gen \
   --versioned-clientset-package "${ANTREA_PKG}/multicluster/pkg/client/clientset/versioned" \
   --listers-package "${ANTREA_PKG}/multicluster/pkg/client/listers" \
   --output-package "${ANTREA_PKG}/multicluster/pkg/client/informers" \
-  --go-header-file hack/boilerplate.go.txt
-
-$GOPATH/bin/controller-gen object:headerFile="hack/boilerplate.go.txt",year=$(date "+%Y") paths="./..."
+  --go-header-file hack/boilerplate/license_header.go.txt
 
+cd multicluster
```

Comment:
Why add this line?

Reply:
The following command is using `paths="./..."`, so we need go to the multi-cluster path. Otherwise it won't work. I will check if `path=./multicluster/...` can work or not.

Reply:
Done